lib/api/src/grpc/proto/points.proto

```diff
@@ -306,6 +306,36 @@ message CountPoints {
   optional bool exact = 3; // If `true` - return exact count, if `false` - return approximate count
 }
 
+message PointsUpdateOperation {
+  message SetPayload {
+      map<string, Value> payload = 1;
+      optional PointsSelector points_selector = 2;
+  }
+  message DeletePayload {
+      repeated string keys = 1;
+      optional PointsSelector points_selector = 2;
+  }
+  message PointStructList {
+    repeated PointStruct points = 1;
+  }
+
+  oneof operation {
+    PointStructList upsert = 1;
+    PointsSelector delete = 2;
+    SetPayload set_payload = 3;
+    SetPayload overwrite_payload = 4;
+    DeletePayload delete_payload = 5;
+    PointsSelector clear_payload = 6;
+  }
+}
+
+message UpdateBatchPoints {
```

Comment:
I derived the name from the other endpoints, which strictly have `<verb><batch>?<noun>`, like `SearchBatchPoints`, `SearchPoints`. 

I agree that `PointsUpdateBatch`, `UpdatePointsBatch` or `BatchUpdatePoints` sound better, but it would go against the current naming scheme.